src/polytax/train.py

```diff
@@ -264,14 +267,16 @@ def decode_and_compute_accuracy(self, x, x_hat):
         # Prepare to decode the labels
         extra_id = 32099
         ids_to_replace_in_preds = []
+
+        # Just curious if it'll be slow since we might iterate the whole thing to get to -100
```

Comment:
yes, it is probably slower than it needs to be, but it runs in a closure asynchronously from the main thread, so it's likely not that urgent.